### PR TITLE
README update [WIP]

### PR DESCRIPTION
**Pull request by johannam999**
### Description

Documentation update after testing Nuka. 
Changes: 
Specified Node.js version to run in order to initiate the demo. 
Information on the default props settings for control buttons (Prev/Next), Paging dots and for Keyboard shortcuts.
Detailed information on ARIA announcement message.

#### Type of Change

Please delete options that are not relevant.

- [x] A documentation update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,9 @@ export default class extends React.Component {
 
 ### Running demo locally
 
-The demo can be launched on your local machine via `webpack-dev-server`. Once you have clone this repo locally, run the following:
+The demo can be launched on your local machine via `webpack-dev-server`. Make sure you are running the node version 9.11 or earlier. Once you have clone this repo locally, run the following:
```
Comment (sarmeyer) [inline README.md]: I know you didn't write this part but I found a typo in `Once you have clone...` it should be `Once you have cloned...`

File: README.md [hunk 2]
```diff
@@ -102,7 +104,7 @@ Or on CodeSandBox at the following url: <a href="https://codesandbox.io/s/04wxlo
 
 A set of eight render props for rendering controls in different positions around the carousel.
 
-- Valid render props for the eight positions are `renderTopLeftControls`, `renderTopCenterControls`, `renderTopRightControls`, `renderCenterLeftControls`, `renderCenterCenterControls`, `renderCenterRightControls`, `renderBottomLeftControls`, `renderBottomCenterControls`, and `renderBottomRightControls`.
+- Valid render props for the eight positions are `renderTopLeftControls`, `renderTopCenterControls`, `renderTopRightControls`, `renderCenterLeftControls`, `renderCenterCenterControls`, `renderCenterRightControls`, `renderBottomLeftControls`, `renderBottomCenterControls`, and `renderBottomRightControls`. The default props are set as `renderCenterLeftControls` for `Previous` button and `renderCenterRightControls` for the `Next` button. To change the position, the default positions need to be disabled by setting them to null.
```
Comment (sarmeyer) [inline README.md]: I would make you addition a separate bullet point so it doesn't get lost in the list of props

File: README.md (hunk 1 above)
Comment (sarmeyer) [reply README.md]: I would also say `...running node version 9.11` and remove the `the`

Comment (johannam999) [reply README.md]: Thanks.I will change that today.

File: README.md (hunk 2 above)
Comment (mariano-formidable) [reply README.md]: Also, don't forget to mention that the `renderBottomCenterControls` are also set to display the "Paging Dots" by default too.

Comment (johannam999) [reply README.md]: @mariano-formidable Good point!

File: README.md [hunk 3]
```diff
@@ -62,6 +62,9 @@ Or on CodeSandBox at the following url: <a href="https://codesandbox.io/s/04wxlo
 | E key                      | Last slide                                          |
 | SpaceBar                   | When `autoplay={true}` pauses and unpauses carousel |
 
+
+- Keyboard shortcuts are disabled as a default. To enable them set the props to `true`.
```
Comment (sarmeyer) [inline README.md]: You might want to add which prop needs to be set to true. I don't think it's part of this section already.

**Pull request merged**